dotcom-rendering/src/components/Accessibility.importable.tsx

```diff
@@ -0,0 +1,78 @@
+import { css } from '@emotion/react';
+import { storage } from '@guardian/libs';
+import { article17, palette } from '@guardian/source/foundations';
+import { useEffect, useState } from 'react';
+import { FrontSection } from './FrontSection';
+
+const formStyle = css`
+	display: flex;
+	flex-direction: column;
+	gap: 0.5rem;
+	padding: 0.5rem 1rem;
+	border: 2px groove ${palette.neutral[86]};
+	margin-left: 10px;
+	${article17}
+`;
+
+const bold = css`
+	font-weight: bold;
+`;
+
+/**
+ * Updates the user's accessibility preferences
+ *
+ * ## Why does this need to be an Island?
+ *
+ * Allows user to select their accessibility preferences
+ */
+export const Accessibility = () => {
+	const [shouldFlash, setShouldFlash] = useState<boolean>(true);
```

Comment:
I think we want to default to `false` until we’ve been able to read the system preferences?